lib/apiservers/engine/backends/container.go

```diff
@@ -667,6 +667,10 @@ func (c *ContainerBackend) ContainerCreate(config types.ContainerCreateConfig) (
 
 	var err error
 
+	if err = checkContainerCount(op, PortLayerClient()); err != nil {
```

Comment:
Should we do similar concurrent control as `checkStorageQuota `?

Reply:
Good catch. Done.